### PR TITLE
Dev/juan

### PR DESCRIPTION
**Pull request by juanc37**
Fixed recent issues

### CODE REVIEW

File: app.js
Comment (tpaulus) [inline app.js:68]: Can we also include Preview Users in the skip list?

**Review by tpaulus — CHANGES_REQUESTED** [review]

File: app.js
Comment (juanc37) [reply app.js:68]: updated in my branch, we just need to test

**Review by tpaulus — APPROVED** [review]
Approved - Provided it is tested before merging.

**Review by tpaulus — APPROVED** [review]

**Comment by juanc37** [comment]
Need to restructure code per toms request

**Pull request merged**
